macOS.md

```diff
@@ -48,6 +45,54 @@ If you are not familiar with video calls, here is a great [article](https://mart
 
 
 
+
+## Checking your computer for Apple Silicon (Apple M1 chips)
+
+If you bought your computer after late 2020, chances are it uses Apple silicon instead of Intel processors. Let's find out...
+
+Open a new Terminal window from Applications > Utilities or search with [Spotlight](https://support.apple.com/en-gb/HT204014):
+
+![](images/open-terminal.png)
+
+Copy-paste the following command in the terminal and hit `Enter` to execute the command.
+
+``` bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/lewagon/setup/master/utils/osx_list_processor_type.sh)"
+```
+
+☝️ The result of the command should indicate whether your computer uses Apple Silicon.
+
+If your computer uses Apple Silicon, proceed with the next section. Otherwise, you can skip to the section below.
```

Comment:
I am a bit confused with the instructions here. Can we insert anchor links?

Reply:
@krokrob I though a solution could be to move the dedicated setup in an emphasised but collapsed paragraph, since the anchors will differ for web and data partials and since the anchor might break in the future...

this behavior with PR https://github.com/lewagon/setup/pull/264 is visible here : https://github.com/lewagon/setup/blob/apple-m1-anchor/macOS.md#checking-your-computer-for-apple-silicon-apple-m1-chips

wdyt ?

Reply:
PERFECT! 👌